engine/battle/ai/scoring.asm

```diff
@@ -223,16 +282,22 @@ AI_Types:
 	pop hl
 	and a
 	jr z, .checkmove
+; discourage (+1)
 	inc [hl]
 	jr .checkmove
 
 .immune
-	call AIDiscourageMove
+	call AIDismissMove
 	jr .checkmove
 
 
 AI_Offensive:
-; Greatly discourage non-damaging moves.
+; Modifiable scores : ALL_MOVES | Randomness : NO
```

Comment:
Not sure what these mean

Reply:
I wasn't sure whether to put these or not. It was a way to describe the general idea of an AI (sub-)layer with the use of description flags. `Modifiable scores` just states which move scores can be modified. `Randomness` is here to tell whether the layer includes randomness in the score modifications.